### PR TITLE
[ja] Updated /content/ja/docs/concepts/workloads/controllers/replicaset.md file

### PR DESCRIPTION
**Pull request by ziyi-xie**
There are several updates in [the original English text](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/controllers/replicaset.md) and therefore this page is outdated.
This PR updated Japanese translation for the [/content/ja/docs/concepts/workloads/controllers/replicaset.md](https://kubernetes.io/ja/docs/concepts/workloads/controllers/replicaset/) file.

Fixes https://github.com/kubernetes/website/issues/36268

### CODE REVIEW

**Comment by ziyi-xie** [comment]
/cc @atoato88 @t-inu 
Hi! I've updated Japanese translation for the [/content/ja/docs/concepts/workloads/controllers/replicaset.md](https://kubernetes.io/ja/docs/concepts/workloads/controllers/replicaset/) file, would you mind checking if everything looks alright at your convenience time please? Here is [the original English text](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/controllers/replicaset.md) and thank you so much!

**Review by Arhell — COMMENTED** [review]
/assign @nasa9084

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -264,6 +261,36 @@ curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/fron
...
+ 3. レプリカ数の多いノード上のPodが、レプリカ数の少ないノード上のPodより優先されます。
+ 4. Podの作成時間が異なる場合、より新しく作成されたPodが古いPodより優先されます(`LogarithmicScaleDown` [フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)が有効の場合、作成時間は整数対数スケールでバケット化されます。)
+
+上記条件のすべてに該当する場合は、ランダム選択となります。
+
+### Pod削除コスト
+
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+
+[`controller.kubernetes.io/pod-deletion-cost`](/docs/reference/labels-annotations-taints/#pod-deletion-cost)  アノテーションを使用すると、ReplicaSetをスケールダウンする際に、どのPodを最初に削除するかについて、ユーザーが優先順位を設定することができます。
```
Comment (nasa9084) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:276]:
```suggestion
[`controller.kubernetes.io/pod-deletion-cost`](/docs/reference/labels-annotations-taints/#pod-deletion-cost)アノテーションを使用すると、ReplicaSetをスケールダウンする際に、どのPodを最初に削除するかについて、ユーザーが優先順位を設定することができます。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -264,6 +261,36 @@ curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/fron
 
 ReplicaSetは、ただ`.spec.replicas`フィールドを更新することによって簡単にスケールアップまたはスケールダウンできます。ReplicaSetコントローラーは、ラベルセレクターにマッチするような指定した数のPodが利用可能であり、操作可能であることを保証します。
 
+スケールダウンする場合、ReplicaSetコントローラーは以下の一般的なアルゴリズムに基づき、利用可能なPodをソートし、スケールダウンするPodの優先順位を付け、削除するPodを選択します:
+ 1. 保留している(またはスケジュール不可な)Podが先にスケールダウンされます。
+ 2. `controller.kubernetes.io/pod-deletion-cost` アノテーションが設定されている場合、値の小さいPodが優先されます。
+ 3. レプリカ数の多いノード上のPodが、レプリカ数の少ないノード上のPodより優先されます。
+ 4. Podの作成時間が異なる場合、より新しく作成されたPodが古いPodより優先されます(`LogarithmicScaleDown` [フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)が有効の場合、作成時間は整数対数スケールでバケット化されます。)
```
Comment (ziyi-xie) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:268]:
```suggestion
 4. Podの作成時間が異なる場合、より新しく作成されたPodが古いPodより優先されます(`LogarithmicScaleDown`[フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)が有効の場合、作成時間は整数対数スケールでバケット化されます。)
```

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 3]
```diff
@@ -264,6 +261,36 @@ curl -X DELETE  'localhost:8080/apis/apps/v1/namespaces/default/replicasets/fron
 
 ReplicaSetは、ただ`.spec.replicas`フィールドを更新することによって簡単にスケールアップまたはスケールダウンできます。ReplicaSetコントローラーは、ラベルセレクターにマッチするような指定した数のPodが利用可能であり、操作可能であることを保証します。
 
+スケールダウンする場合、ReplicaSetコントローラーは以下の一般的なアルゴリズムに基づき、利用可能なPodをソートし、スケールダウンするPodの優先順位を付け、削除するPodを選択します:
+ 1. 保留している(またはスケジュール不可な)Podが先にスケールダウンされます。
+ 2. `controller.kubernetes.io/pod-deletion-cost` アノテーションが設定されている場合、値の小さいPodが優先されます。
```
Comment (ziyi-xie) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:266]:
```suggestion
 2. `controller.kubernetes.io/pod-deletion-cost`アノテーションが設定されている場合、値の小さいPodが優先されます。
```

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 4]
```diff
@@ -313,4 +340,10 @@ ReplicaSetは[_ReplicationControllers_](/docs/concepts/workloads/controllers/rep
 この2つは、ReplicationControllerが[ラベルについてのユーザーガイド](/ja/docs/concepts/overview/working-with-objects/labels/#label-selectors)に書かれているように、集合ベース(set-based)のセレクター要求をサポートしていないことを除いては、同じ目的を果たし、同じようにふるまいます。  
 このように、ReplicaSetはReplicationControllerよりも好まれます。
 
+## {{% heading "whatsnext" %}}
 
+* [Pod](/ja/docs/concepts/workloads/pods/)について学ぶ。
+* [Deployments](/ja/docs/concepts/workloads/controllers/deployment/)について学ぶ。
```
Comment (t-inu) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:346]:
```suggestion
* [Deployment](/ja/docs/concepts/workloads/controllers/deployment/)について学ぶ。
```

**Review by t-inu — COMMENTED** [review]
I left some comments.

File: content/ja/docs/concepts/workloads/controllers/replicaset.md [hunk 5]
```diff
@@ -296,11 +323,11 @@ ReplicaSetは単独で使用可能ですが、現在では、ReplicaSetは主に
 
 ユーザーがPodを直接作成するケースとは異なり、ReplicaSetはNodeの故障やカーネルのアップグレードといった破壊的なNodeのメンテナンスなど、どのような理由に限らず削除または停止されたPodを置き換えます。
 このため、我々はもしユーザーのアプリケーションが単一のPodのみ必要とする場合でもReplicaSetを使用することを推奨します。プロセスのスーパーバイザーについても同様に考えると、それは単一Node上での独立したプロセスの代わりに複数のNodeにまたがった複数のPodを監視します。
-ReplicaSetは、Node上のいくつかのエージェント(例えば、KubeletやDocker）に対して、ローカルのコンテナ再起動を移譲します。
+ReplicaSetは、Node上のいくつかのKubeletのようなエージェントに対して、ローカルのコンテナ再起動を移譲します。
```
Comment (atoato88) [inline content/ja/docs/concepts/workloads/controllers/replicaset.md:326]:
```suggestion
ReplicaSetは、KubeletのようなNode上のいくつかのエージェントに対して、ローカルのコンテナ再起動を移譲します。
```
Your translation is also OK, but my suggestion is good for avoiding some ambiguities, I think.

**Review by atoato88 — COMMENTED** [review]
Please check mu comments.

**Comment by atoato88** [comment]
/lgtm
Thank you for quick update.

**Comment by t-inu** [comment]
/label tide/merge-method-squash
LGTM

**Pull request merged**